module/applications/compendium-browser.mjs

```diff
@@ -325,56 +560,127 @@ export default class CompendiumBrowser extends HandlebarsApplicationMixin(Applic
   async _prepareResultsContext(context, options) {
     // TODO: Determine if new set of results need to be fetched, otherwise use old results and re-sort as necessary
     // Sorting changes alone shouldn't require a re-fetch, but any change to filters will
-    this.#results = CompendiumBrowser.fetch(
-      CONFIG[context.filters.documentClass].documentClass,
-      {
-        types: context.filters.types,
-        filters: CompendiumBrowser.applyFilters(context.filterDefinitions, context.filters.additional)
-      }
-    );
+    const filters = CompendiumBrowser.applyFilters(context.filterDefinitions, context.filters.additional);
+    // Add the name filter
+    if ( this.#filters.name?.length ) filters.push({ k: "name", o: "istartswith", v: this.#filters.name });
```

Comment:
Do we want to use an `icontains` filter here instead, so searching for something like "fireball" will return both "Wand of Fireballs" and "Necklace of Fireballs" or searching for "warning" will produce all of the weapons of warning?

Reply:
Yeah you're probably right. It would be a bit silly if you couldn't search for 'dragon' and get all the dragons, for example.